Twitch EventSub library/EventSubSocketWrapper.cs

```diff
@@ -53,29 +52,35 @@ public class EventSubSocketWrapper : IEventSubSocketWrapper
         private async Task OnServerSideTerminationAsync(object sender, string e)
         {
             _logger.LogInformation(e,sender);
-            _connectionActive = false;
             await OnOutsideDisconnectAsync.TryInvoke(this, e);
+            _socket.Dispose();
         }
 
-        public async Task<bool> ConnectAsync(string connectUrl = DefaultWebSocketUrl)
+        public async Task<bool> ConnectAsync(Uri url = null)
         {
+            _connectionActive = _socket?.IsRunning ?? false;
```

Comment:
Instead of assigning `_connectionActive` and maintaining the flag status, you can create auto-property that will be self-maintained:
`private bool IsConnectionActive => _socket?.IsRunning ?? false;`

Reply:
Done.